src/devices/video/ppu2c0x_vt.h

```diff
@@ -0,0 +1,78 @@
+// license:BSD-3-Clause
```

Comment:
This header is missing `#include` guards (the `#ifndef ... #define ... #pragma once` stuff).  Could you please add it?